funcx_web_service/routes/funcx.py

```diff
@@ -5,7 +5,6 @@
 
 from flask import current_app as app, Blueprint, jsonify, request, abort, send_from_directory, g
 
-from funcx.utils.errors import RegistrationError
```

Comment:
I don't think its great to be importing error types from funcx SDK, service should have its own error types